hardhat.config.mainnet_fork.ts

```diff
@@ -30,8 +30,8 @@ const config: HardhatUserConfig = {
     networks: {
         mainnet_fork: {
             live: false,
-            url: env.HYPERDRIVE_ETHEREUM_URL ?? "http://anvil:8545",
-            accounts: [env.DEPLOYER_PRIVATE_KEY ?? DEFAULT_PK],
+            url: env.RPC_URL ?? "http://anvil:8545",
```

Comment:
Why didn't this change in all of the config files?

Reply:
Good catch, was initially considering updating some/all of the environment variables to match what is used in `hyperdrive-infra`, however when going through it plus the `hyperdrive-infra` integration it seemed sizeable enough to do in a separate PR.

Reverted to what it was originally (as well as updated clone config to match)